components/EmotionForm.js

```diff
@@ -166,10 +133,10 @@ const StyledEmotionForm = styled.form`
   border-radius: 8px;
 `;
 
-const StyledLabel = styled.label`
-  color: ${(props) => (props.$hasError ? "#ff0000" : "#000000")};
-  font-weight: ${(props) => (props.$hasError ? "bold" : "normal")};
-`;
+// const StyledLabel = styled.label`
+//   color: ${(props) => (props.$hasError ? "#ff0000" : "#000000")};
+//   font-weight: ${(props) => (props.$hasError ? "bold" : "normal")};
+// `;
```

Comment:
You have a lot of comments in your code. We were told by the coaches to delete all comments before posting the PR.

Reply:
Thanks! We deleted all of them.